src/notation/view/widgets/editstyle.ui

```diff
@@ -7046,7 +7046,7 @@ By default, they will be placed such as that their right end are at the same lev
         <item>
          <widget class="QGroupBox" name="groupBox_slursTies">
           <property name="title">
-           <string>Slurs/Ties</string>
+           <string>Slurs &amp;&amp; ties</string>
```

Comment:
Double ampersand was necessary here to avoid creating a mnemonic.

Reply:
@cbjeukendrup wasn't right. "DOUBLE ampersand was necessary here" 😂🤣
Ampersand evolution:
& -> &amp; -> &amp;&amp; 😂🤣